docs/platforms/python/migration/1.x-to-2.x.mdx

```diff
@@ -61,26 +55,92 @@ The API function `last_event_id()` was removed. The last event ID is still retur
 
 ## Custom Instrumentation
 
-You can no longer mutate a manually-created transaction in a `configure_scope` block. Instead, you need to get the current scope to mutate the transaction. Here is a recipe on how to change your code to make it work:
+Hub-based APIs as well as some scope-based APIs like `configure_scope()` and `push_scope()` are now deprecated as a consequence of [reworking](/platforms/python/enriching-events/scopes/) the way the SDK saves and propagates data internally. Instead, SDK 2.0 introduces the concept of a global, isolation and current scope. In short:
 
-<SignInNote />
+- The **global scope** holds data that doesn't change as long as your app is running, for instance the current release.
+- The **isolation scope** follows a single basic unit of your program such as a single request-response cycle or one task in a task queue and holds data pertaining to it. In most cases, the lifecycle of an isolation scope maps to the lifecycle of a transaction.
+- The **current scope** wraps a smaller part of the code, often something that's being tracked by a single span.
+
+The SDK will take care of deciding what data belongs on which scope as long as you use the top-level API, so for instance `sentry_sdk.set_tag()` as opposed to `scope.set_tag()`.
```

Comment:
```suggestion
The SDK will take care of deciding what data belongs on which scope as long as you use the top-level API. For example, you can use `sentry_sdk.set_tag()` and let the SDK handle the data for you, instead of manually setting it with `scope.set_tag()`.
```
I thought it could help to explain more for the user here. Hope I interpreted it correctly.